### PR TITLE
Remove unused greenwave database from ci.

### PR DESCRIPTION
**Pull request by sebwoj**
Signed-off-by: Sebastian Wojciechowski <s.wojciechowski89@gmail.com>

### CODE REVIEW

File: devel/ci/integration/tests/fixtures/greenwave.py [hunk 1]
```diff
@@ -24,19 +24,16 @@
...
     Args:
         docker_backend (conu.DockerBackend): The Docker backend (fixture).
         docker_network (str): The Docker network ID (fixture).
-        db_container(conu.DockerContainer): The PostgreSQL container (fixture).
 
     Yields:
         conu.DockerContainer: The Greenwave container.
     """
-    # Prepare the database
-    make_db_and_user(db_container, "greenwave")
```
Comment (bowlofeggs) [inline devel/ci/integration/tests/fixtures/greenwave.py:39]: Ah interesting, I didn't realize that Greenwave didn't use a DB. I went and took a peek at their settings and I see that they are not. I also see that we don't download a Greenwave db dump.

**Review by bowlofeggs — APPROVED** [review]
Thanks!

**Comment by sebwoj** [comment]
It looks like quay.io/repository/factory2/waiverdb and quay.io/repository/factory2/greenwave are unreachable.

**Comment by bowlofeggs** [comment]
Yeah quay.io doesn't seem to be the most reliable thing around. I've got an open pull request that should retry on build failures: #3156.

**Comment by bowlofeggs** [comment]
I'm applying this patch to fix a flake8 error:

```
diff --git a/devel/ci/integration/tests/fixtures/greenwave.py b/devel/ci/integration/tests/fixtures/greenwave.py
index 8801b5d0..3c80191f 100644
--- a/devel/ci/integration/tests/fixtures/greenwave.py
+++ b/devel/ci/integration/tests/fixtures/greenwave.py
@@ -20,8 +20,6 @@ import conu.utils.probes
 import docker.errors
 import pytest
 
-from ..utils import make_db_and_user
-
 
 @pytest.fixture(scope="session")
 def greenwave_container(docker_backend, docker_network):
```

**Pull request merged**